### DISCUSSION TITLE
Github Sponsor Payout never issues (first-time new user of Github Sponsor)

### DISCUSSION THREAD

**Discussion by jhcarl0814** [community/community#72298] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I got a one-time sponsorship on 2023-08-17 and a monthly sponsorship starting from 2023-02. But the payment is stuck:

In Github Sponsor dashboard it says:
```
ESTIMATED NEXT PAYOUT [month] 22
```
and whenever that day comes, it becomes same day of the next month:
```
ESTIMATED NEXT PAYOUT [next month] 22
```

In Stripe page it says:
```
Payout expected on 22 [month]
```
and whenever that day comes, it becomes same day of the next month:
```
Payout expected on 22 [next month]
```
----------
## Is there a place in Github Sponsor dashboard or Stripe page that tells me why it never issues payout, and what actions should I take to fix this?

**Comment by ktauchathuranga** [comment]
i advice you to open a ticket in GitHub support and letting them know the situation
use this link to contact support [GitHub Support](https://support.github.com/request)

**Comment by DHayes10** [comment, marked as answer]
Hi @jhcarl0814, 

We appreciate you reaching out about the issue of not receiving your monthly payouts. I have contacted Stripe on your behalf. The payment is in transit and due to arrive in your bank account on November 8th. 

Thank you, 
GitHub Sponsors

**Reply by tobySolutions** [reply]
> Hi @jhcarl0814,
> 
> We appreciate you reaching out about the issue of not receiving your monthly payouts. I have contacted Stripe on your behalf. The payment is in transit and due to arrive in your bank account on November 8th.
> 
> Thank you, GitHub Sponsors

Hello there @DHayes10, this same issue happened to me just now. I saw on my Stripe Dashboard that a payout has been completed, but, I didn't get the payout in my local account at all. It's really weird. Is there a way you can help with this?

**Reply by rashidpathiyil** [reply]
Have you solved the issue @devnshankar ?

**Reply by cuilei5205189** [reply]
Hi @DHayes10,

I’m experiencing a similar GitHub Sponsors payout issue.

My Sponsors dashboard estimated payout has rolled over to the next month, and in Stripe Express the balance is already available, but there is no payout pending or in transit.

I have already opened a GitHub Support ticket: #4508632, but I haven’t received a response yet.

Could you please help check this or route it to the GitHub Sponsors team? It looks like the payout was not initiated rather than delayed by the bank.

Thank you.

> Hi @jhcarl0814,
> 
> We appreciate you reaching out about the issue of not receiving your monthly payouts. I have contacted Stripe on your behalf. The payment is in transit and due to arrive in your bank account on November 8th.
> 
> Thank you, GitHub Sponsors

**Comment by mrwtheboss** [comment]
I have exactly the same issue

**Comment by mabaega** [comment]
Hi, I am also getting the same issue,

on stripe it shows
Available Payout : xxxxx

Github Sponsor Dashboard shows
Estimated Next Payout
xxxxx
Aug 22

Now it is the 23rd, and nothing has happened, please explain.
Thanks.

**Reply by rogerissimo1975-hue** [reply]
Cant live whitout any money

**Comment by vczb** [comment]
Same here! I will stop using Github sponsor because of this issue. I'm waiting more than 1 year to payout and nothing happen

**Reply by irsyadadl** [reply]
Same here.

**Reply by leandromoh** [reply]
Same here

**Comment by dromer** [comment]
Same, the "Estimated Next Payout" just keeps moving to the following month and no payout is ever initiated.

**Comment by carlosedp** [comment]
Same issue here, never received a payout and the dates keep rolling.

Any help @DHayes10 ?

**Comment by preeti13456** [comment]
I'm also facing the same issue kindly pls help

**Comment by preeti13456** [comment]
SAME HERE
________________________________

**Comment by thinh-vu** [comment]
I am beyond frustrated and frankly, desperate, at this point. I have no idea where else to turn. My GitHub Sponsors payouts have completely stopped since July 22nd.
For months, everything worked perfectly. Now, my payouts for August and September are missing. When I check my Stripe Connect account, it shockingly says my payouts are now "Manually paid out," a change I never requested or was notified about.
There's no information, no emails, no warnings. My dashboard just shows the next payout is now inexplicably pushed to October 22nd, with no guarantee it will even happen.
I'm attaching two screenshots:
1. My GitHub Sponsors dashboard showing the last payout was on July 22nd and the current pending amount.
<img width="1792" height="607" alt="pending-payouts" src="https://github.com/user-attachments/assets/828691cc-cef5-408b-ba12-61460fe81c07" />

2. My Stripe Connect dashboard showing the "Manually paid out" status.
<img width="671" height="335" alt="manual-paid-out" src="https://github.com/user-attachments/assets/b2091256-707d-42de-b55b-095f52b59469" />

Can someone from GitHub please tell me what is going on? @DHayes10 Please help!
Username: thinh-vu

**Comment by lumirlumir** [comment]
What's the root cause of this problem or bug? It's still happening in 2026, and I haven't received my sponsorship money for about six months.

**Comment by preeti13456** [comment]
same
________________________________
